packages/design-systems/src/components/composite/Datagrid/SearchFilter/FilterRow.tsx

```diff
@@ -152,20 +164,80 @@ export const FilterRow = <TData extends Record<string, unknown>>(
     return selectedColumnObject?.meta?.type || "text";
   }, [selectedColumnObject?.meta?.type]);
 
+  const renderValueInput: JSX.Element = useMemo(() => {
```

Comment:
細かいのですが、ここで `useMemo` すると予期しないキャッシュが働いてしまう可能性があるので、JSX.Elementに対してはuseMemoは使いたくないです。

パフォーマンスの問題が顕在化してからまた考えるで良いと思います。

```suggestion
  const renderValueInput = () => {
```